vocab/constants/VOCAB_QUDT-CONSTANTS-v2.1.ttl

```diff
@@ -2252,10 +2252,11 @@ constant:PermeabilityOfVacuum
 .
 constant:PermittivityOfVacuum
   a qudt:PhysicalConstant ;
-  dcterms:description "\"Permittivity of Vacuum}, also known as the \\textit{electric constant}, is a constant whose value is \\SI{\\approx 6.854188e-12}{\\farad\\per\\metre\"."^^rdf:HTML ;
+  dcterms:description "\"Permittivity of Vacuum}, also known as the \\textit{electric constant}, is a constant whose value is \\SI{\\approx 6.854188e-12}{\\farad\\per\\metre. Sometimes also referred to as the Permittivity of Free Space.\""^^rdf:HTML ;
```

Comment:
The \\SI LaTeX needs to be changed because the SI library is not in use any more.

Needs to be:

terms:description \(\\textit{Permittivity of Vacuum}\), also known as the \(\\textit{electric constant}\) is a constant whose value is \(\\approx 6.854188e-12\\,\\farad\\,\\per\\,metre\). Sometimes also referred to as the Permittivity of Free Space.\""^^qudt:LatexString ;

Reply:
I thought you were fixing all the latex issues!

Reply:
Made the change, but see my comment on your next comment.

Reply:
Agree to deferring the fix